test/integration/cdi/example-driver/app/server.go

```diff
@@ -44,6 +44,7 @@ import (
 	"k8s.io/component-base/metrics/legacyregistry"
 	"k8s.io/component-base/term"
 	"k8s.io/klog/v2"
+	plugin "k8s.io/kubernetes/test/integration/cdi/example-driver/kubeletplugin"
```

Comment:
Please refactor the code:
- generic functionality belongs under "example-driver" (like "example-driver/controller") and will be moved into a staging repo later
- the driver code itself is under "example-driver/app"

Reply:
maybe done.!